e2e/release/src/multiple-release-branches.test.ts

```diff
@@ -124,41 +124,41 @@ describe('nx release multiple release branches', () => {
 
     expect(initialVersionResult).toMatchInlineSnapshot(`
 
-      NX   Running release version for project: {project-name}
+      NX   Running release version for project: {project-name-1}
```

Comment:
I removed this, however it belongs to the fix of #27186. Those changes allowed me to figure out why the tests sometimes fail: with a different order (instead of 1, 2, 3), the output is slightly different (the braces are placed in a different place in the test `git-tag version resolver should detect tags in other branches if none are reachable from the current commit`).

Reply:
What do you mean by "it belongs to the fix" sorry? Are you saying it is required for it to be functional/effective? Or purely that it was useful while troubleshooting?

Reply:
It was useful while troubleshooting. However, it might also be useful in the future to more easily see what is happening if it fails again in the future.

The E2E tests use three test packages, `my-pkg-1` to `my-pkg-3`. Then one of the packages is updated and an nx release is run. In the output, the changed package has a slightly different diff, with an additional `}` and `+`. Since there is just the output string, the expectation is that it contains the output for `my-pkg-1`, then `my-pkg-2`, then `my-pkg-3`, with the additional `}` at the changed package.

When the bug happened, it was because the order of the packages was different, ex. 2 => 3 => 1. The additional text was at the correct package, but that package was in a different place in the output string.

Technically the order shouldn't matter. Unfortunately, due to the limitations of a string comparison, it does. So you could

1. just leave it as is
2. add the package-numbering to the output, to see when the ordering is different than expected
3. clean up the output to remove the extra `}` and `+`, so the ordering doesn't matter (since only the changed version number is what is relevant in the test)